luacompiler/src/lib/bytecode/instructions.rs

```diff
@@ -40,6 +40,7 @@ pub fn format_instr(instr: u32) -> String {
         10 => "Exp",
         11 => "GetAttr",
         12 => "SetAttr",
+        13 => "Closure",
```

Comment:
Why are we using the term "closure" whereas I thought there were only functions?

Reply:
Functions can also modify local variables that were defined in outer scopes. From this point of view, they are closures. Also, the lua implementation translates functions to closures.

Reply:
OK. [I'm not sure the terminological shift is useful, but it doesn't hurt much either. In most languages, I see functions and closures as synonyms from a normal user's perspective.]